source/iostreams.tex

```diff
@@ -14713,6 +14715,20 @@
 by \ref{fs.path.cvt}.
 \end{itemdescr}
 
+\indexlibrarymember{display_string}{path}%
+\begin{itemdecl}
+std::string display_string() const;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+\tcode{format("{}", *this)}.
+\begin{note}
+The returned string is suitable for use with formatting \iref{format.functions}
+and print functions \iref{print.fun}.
```

Comment:
There should be no space before `\iref`.

Reply:
Fixed the whitespace in the last commit and force-pushed.